lib/tasks/evm_application.rb

```diff
@@ -102,11 +102,12 @@ def self.output_workers_status(servers)
            w.miq_server_id,
            w.queue_name || w.uri,
            w.started_on && w.started_on.iso8601,
-           w.last_heartbeat && w.last_heartbeat.iso8601]
+           w.last_heartbeat && w.last_heartbeat.iso8601,
+           (w.proportional_set_size || w.memory_usage || 0) / 1.megabyte]
```

Comment:
> Is it pedantic?

Well, I said it was so it is!

But really, my rational is that seeing a `0 Mb` process in the status might be confusing to a user, and might give the impression of "the worker is just super efficient memory!" to someone who doesn't know what is going on under the hood.  But again, I do think I am being pedantic, and problem more (questionably correct) info is better than none.